Makefile

```diff
@@ -94,6 +94,7 @@ clean:
 	rm -f $(MICROSERVICES)
 
 test:
+	if which hadolint > /dev/null ; then hadolint --config hadolint.yml `find * -type f -name 'Dockerfile*' -print` ; elif which docker > /dev/null ; then docker run --rm -v `pwd`:/host:ro,z --entrypoint /bin/hadolint hadolint/hadolint:latest --config /host/.hadolint.yml `find * -type f -name 'Dockerfile*' | xargs -i echo '/host/{}'` ; fi
```

Comment:
ci failed on this line:

```Bash
hadolint: hadolint.yml: openBinaryFile: does not exist (No such file or directory)
```

I think it needs to be `hadolint --config .hadolint.yml`